### PR TITLE
Fix deployment extraEnvs indentation in operator chart

### PR DESCRIPTION
**Pull request by DaDummy**
Fix bad operator extraEvns indentation by matching the statement to how other lists are expanded in the deployment template.

The previous implementation caused a yaml parsing error due to the expanded extraEnvs list being indented by more than 8 spaces while the other list entries from the template just have 8 spaces indentation.

### CODE REVIEW

File: charts/postgres-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ spec:
           value: {{ template "postgres-operator.controllerID" . }}
       {{- end }}
       {{- if .Values.extraEnvs }}
-      {{- .Values.extraEnvs | toYaml | nindent 8 }}
+{{ toYaml .Values.extraEnvs | nindent 8 }}
```
Comment (FxKu) [inline charts/postgres-operator/templates/deployment.yaml:57]: why is it nindent and not just indent?

Comment (DaDummy) [reply charts/postgres-operator/templates/deployment.yaml]: I missed that the comparable lines in the same file all used indent. I've adjusted the line now to use indent, too.

**Comment by FxKu** [comment]
:+1:

**Comment by hughcapet** [comment]
👍

**Pull request merged**